src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -71,12 +73,15 @@ public long id() {
         throw new UnsupportedOperationException("id() not yet implemented");
     }
 
-    // @todo #15:30min Implement time() by parsing the string representation
-    //  of transaction according to the pattern, described in the white
-    //  paper. Replace relevant test case with actual tests.
     @Override
-    public ZonedDateTime time() {
-        throw new UnsupportedOperationException("time() not yet implemented");
+    public ZonedDateTime time() throws IOException {
+        return new ZonedDateTimeOf(
```

Comment:
@carlosmiranda even though by default `ZonedDateTimeOf` will use ISO ISO-8601, I think it may by better, to perfectly match the spec, to express it explicitly here.
It's up to discussion of course :)